drivers/video/video_device.h

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2024 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_VIDEO_DEVICE_H_
+#define ZEPHYR_INCLUDE_VIDEO_DEVICE_H_
+
+#include <zephyr/device.h>
+
+struct video_device {
+	int ind;
+	const struct device *dev;
+};
+
+int video_register_vdev(struct video_device *vdev, const struct device *dev);
```

Comment:
could it simply be `video_device_register/unregister()`?

Reply:
I followed the naming pattern with the prefix `video_dosomething()` but yes we can simplify it as your suggestion.